src/ring_theory/witt_vector/mul_p.lean

```diff
@@ -0,0 +1,78 @@
+/-
+Copyright (c) 2020 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin
+-/
+
+import ring_theory.witt_vector.is_poly
+
+/-!
+## Multiplication by `n` in the ring of Witt vectors
+
+In this file we show that multiplication by `n` in the ring of Witt vectors
+is a polynomial function. We then use this fact to show that the composition of Frobenius
+and Verschiebung is equal to multiplication by `p`.
+
+### Main declarations
+
+* `mul_n_is_poly`: multiplication by `n` is a polynomial function
+
+-/
+
+namespace witt_vector
+
+variables {p : ℕ} {R : Type*} [hp : fact p.prime] [comm_ring R]
+local notation `𝕎` := witt_vector p -- type as `\bbW`
+
+local attribute [semireducible] witt_vector
```

Comment:
On my machine, removing this line approximately halves the compilation time. (And it seems to not break anything.)
```suggestion
```

Reply:
Aaah cool! This file indeed only uses the API that we've built, and shouldn't need to look under the hood. Thanks for catching this!